challenge_restaurant/src/challenge_restaurant/take_orders.py

```diff
@@ -66,20 +69,18 @@ def execute(self, userdata=None):
             rospy.loginfo('nr_tries: %d', nr_tries)
 
             self._robot.speech.speak("What would you like to order?")
+            self._robot.speech.speak("Please speak fast without breaks.")
             count = 0
             while not rospy.is_shutdown():
                 count += 1
 
                 try:
                     speech_result = self._robot.hmi.query(description="Can I please take your order",
-                                                          grammar=knowledge.order_grammar, target="O")
+                                                        grammar=knowledge.order_grammar, target="O")
                     break
                 except TimeoutException:
-                    if count < 5:
-                        self._robot.speech.speak(random.choice(["I'm sorry, can you repeat",
-                                                                "Please repeat your order, I didn't hear you",
-                                                                "I didn't get your order, can you repeat it",
-                                                                "Please speak up, as I didn't hear your order"]))
```

Comment:
I would prefer this option as it makes the robot not repeat itself. I get the choice for the single command in a robocup scenario but I prefer this one for a general waiter-robot